### PR TITLE
Remove unnecessary columns from all pages and from footer

### PR DESCRIPTION
**Pull request by rosalieper**
Bug: T350370

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -4,11 +4,9 @@
       <router-view v-if="customLayout"/>
       <v-container v-else class="full-height-content">
         <v-row >
-          <v-col cols="1"></v-col>
-          <v-col cols="10">
+          <v-col>
```
Comment (m90) [inline src/App.vue:7]: Idea: if we now have a row that contains a single column, there is no need for a grid system anymore, and we could also just remove both the `v-row` and the `v-col` entirely with the `v-container` making sure everything stays in place.

**Review by m90 — COMMENTED** [review]
This works as described in the ticket. I left a suggestion about how to further simplify this, lmk what you think.

File: src/App.vue (hunk 1 above)
Comment (rosalieper) [reply src/App.vue:7]: I am leaving container in this case because of the class.

File: src/components/Layout/Foot.vue [hunk 2]
```diff
@@ -1,10 +1,6 @@
 <template>
   <footer class="footer">
   <v-footer color="primary lighten-1">
-    <v-container>
```
Comment (m90) [inline src/components/Layout/Foot.vue:4]: I think the container is still needed for giving the footer content a width, it's just `col` and `row` that are superfluous.

image

**Review by m90 — APPROVED** [review]
:sparkles:

**Pull request merged**